### PR TITLE
Feature/confirmationPage@ReactStrap

### PR DESCRIPTION
**Pull request by Inhockorea**
1) Confirmation Page
2) Integrated bit of Material UI 
3) Passes Data from /editentity to confirmation 
4) ReactStrap built

Material UI would be in separate branch // this is a stable react strap branch

### CODE REVIEW

**Comment by Inhockorea** [comment]
Left Navigation component here at the moment for now( since waiting for other navbar merge request to be completed)
Also, Ian gave me an advice to work on an individual component, not at the container level (so having a navbar in the dashboard level can be helpful and can be deleted anytime)

File: Dashboard/src/Entity.js [hunk 1]
```diff
@@ -1,12 +1,48 @@
...
+import React, { useEffect, useState } from "react"
+import { useHistory } from "react-router-dom"
+import { makeStyles } from "@material-ui/core/styles"
+import Button from "@material-ui/core/Button"
+import Alert from "@material-ui/lab/Alert"
 import { Styles } from "./ReactTable/AdminDashboardStyle"
 import AdminDashboard from "./ReactTable/AdminDashboard"
 
+// Styling used for MaterialUi
+const useStyles = makeStyles(() => ({
```
Comment (i-mckinney) [inline Dashboard/src/Entity.js:10]: Same as the other PR - Lets go over this tomorrow.

File: Dashboard/src/EntityDetails/EditEntity.js [hunk 2]
```diff
@@ -7,12 +9,42 @@ import { Styles } from "./EditEntityStyle"
 import EntityTableCell from "./EntityTableCell"
 import EntityTable from "./EntityTable"
 
+// Styling used for MaterialUi
+const useStyles = makeStyles(() => ({
```
Comment (i-mckinney) [inline Dashboard/src/EntityDetails/EditEntity.js:13]: Just like before :)

File: Dashboard/src/routes-nav/Navigation.js [hunk 3]
```diff
@@ -1,51 +1,94 @@
...
-  NavbarBrand,
-  Nav,
-  NavItem,
-  NavLink,
-} from "reactstrap"
+  BottomNavigation,
+  BottomNavigationAction,
+  makeStyles,
+} from "@material-ui/core"
+import Computer from "@material-ui/icons/Computer"
```
Comment (i-mckinney) [inline Dashboard/src/routes-nav/Navigation.js:8]: import { Computer as Computer, CreditCard as CreditCard, etc.} from '@material-ui/icons' <- see if this works. If not lets introduce a babel config edit towards the bottom of the page here. https://material-ui.com/guides/minimizing-bundle-size/

File: Dashboard/src/routes-nav/Navigation.js [hunk 4]
```diff
@@ -1,51 +1,94 @@
...
+import CreditCard from "@material-ui/icons/CreditCard"
+import AccountBalance from "@material-ui/icons/AccountBalance"
+import helixLogo from "../helixLogo.jpg"
 
-/** @returns {jsx} Handles Navigation for Helix Regulatory DashBoard. */
-const Navigation = () => {
-  // When the browser window gets to small, nav bar collapses and toggle is used to show and hide the navbar.
-  const [isOpen, setIsOpen] = useState(false)
-  const toggle = () => setIsOpen(!isOpen)
+const useStyles = makeStyles({
```
Comment (i-mckinney) [inline Dashboard/src/routes-nav/Navigation.js:15]: I'm commenting these so we can easily find them later.

File: Dashboard/src/routes-nav/Navigation.js [hunk 5]
```diff
@@ -1,51 +1,94 @@
...
+    <div>
+      <div className={classes.root}>
+        <button
+          onClick={handleHome}
+          type="button"
+          className={classes.homeButton}
+        >
+          <img src={helixLogo} alt="helixLogo" />
+        </button>
+        <BottomNavigation
```
Comment (i-mckinney) [inline Dashboard/src/routes-nav/Navigation.js:56]: Lets use bottom nav for a footer component (haven't really talked about that yet). Check out AppBar https://material-ui.com/components/app-bar/

**Pull request closed without merging**